test/csharp/TestOnnxRuntimeGenAIAPI.cs

```diff
@@ -5,28 +5,24 @@
 using System.IO;
 using System.Linq;
 using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
 using Xunit;
 using Xunit.Abstractions;
 
 namespace Microsoft.ML.OnnxRuntimeGenAI.Tests
 {
-    public class TestsFixture : IDisposable
```

Comment:
This test-case level fixture doesn't work. Because after `OgaShutdown` is called, it can't recover and the entire test crashes.